src/devices/GamepadMappings.js

```diff
@@ -115,7 +116,7 @@ let openVr = {
 
 let windowsMixedReality = {
```

Comment:
Not sure these are correct?  The gamepad Ids of the default Microsoft controllers is different than those of the Samsung Odyssey.

Reply:
Do you happen to know the gamepad and/or display id for the Odyssey? I don't have one on hand to check.

Reply:
Hey, so it turns out the A-Frame crew also have a lot of these IDs in their code base. ;)